doc/2/api/controllers/security/search-api-keys/index.md

```diff
@@ -12,14 +12,23 @@ Searches for a user API keys.
 To search for an API key corresponding to a token you can search on the `fingerprint` property which is a SHA256 hash of the token.
 :::
 
+<SinceBadge version="change-me"/>
```

Comment:
Are we tracking these things effectively so they don't get diluted in the soup?

Reply:
The release tool automatically update the badges with the released version